src/scripts/record/reactCommitsAndUserLogs.js

```diff
@@ -0,0 +1,195 @@
+const { parentPort } = require('worker_threads');
+const net = require('net');
+const fs = require('fs');
+const path = require('path');
+
+const reactFilePath = path.join(
+  __dirname,
+  '../../../../../../data/reactCommits.txt'
+);
+const userLogsFilePath = path.join(
+  __dirname,
+  '../../../../../../data/userLogs.txt'
+);
```

Comment:
```suggestion
const reactFilePath = path.join(
  __dirname,
  '../../../../../../data/reactCommits.txt'
);
const userLogsFilePath = path.join(
  __dirname,
  '../../../../../../data/userLogs.txt'
);
```

Can we use absolute path instead of relative.